gtk4-macros/Cargo.toml

```diff
@@ -19,8 +19,8 @@ xml_validation = ["quick-xml"]
 
 [dependencies]
 anyhow = "1.0"
-quick-xml = {version = "0.22.0", optional = true}
-proc-macro-crate = "1.0"
+quick-xml = {version = "0.25", optional = true}
```

Comment:
However this here is necessary: 0.22 and 0.25 are not considered compatible so cargo won't automatically update it (and as you saw you need to apply some source changes too).